foundry/foundry.py

```diff
@@ -854,9 +852,52 @@ def toTorch(self, raw=None, split=None):
                 for key in df.keys():
                     arr.append(df[key].values)
 
-            return FoundryDataset_Torch(inputs, targets)
+            return FoundryDatasetAsTorchDataset(inputs, targets)
         else:
             raise NotImplementedError
+
+    def to_tensorflow(self, split=None):
+        """Convert Foundry Dataset to a Tensorflow Sequence
+
+        Arguments:
+            split (string): Split to create Tensorflow Sequence on.
+                    **Default:** ``None``
+
+        Returns: (FoundryDatasetAsTensorflowSequence) Tensorflow Sequence of all the data from the specified split
+
+        """
+        raw = self.load_data(as_hdf5=False)
+        
+        if not split:
+            split = self.dataset.splits[0].type
+
+        if self.dataset.data_type.value == "hdf5":
+            inputs = []
+            targets = []
+            for key in self.dataset.keys:
+                if len(raw[split][key.type][key.key[0]].keys()) != self.dataset.n_items:
+                    continue
+
+                val = np.array([raw[split][key.type][key.key[0]][k] for k in raw[split][key.type][key.key[0]].keys()])
+                if key.type == 'input':
+                    inputs.append(val)
+                else:
+                    targets.append(val)
+
+            return FoundryDatasetAsTensorflowSequence(inputs, targets)
+        elif self.dataset.data_type.value == "tabular":
+            inputs = []
+            targets = []
+
+            for index, arr in enumerate([inputs, targets]):
+                df = raw[split][index]
+                for key in df.keys():
+                    arr.append(df[key].values)
+
+            return FoundryDatasetAsTensorflowSequence(inputs, targets)
+        else:
+            raise NotImplementedError
```

Comment:
👏🏼 Good to have a "default" action that alerts the user to problems with an exception.

Reply:
I'm not quite sure what this means.

Reply:
That the if/elif statement throws an error if the code ends up matching a condition you didn't expect when implementing it. The "default" behavior is the term for such a condition in C/C++/Java.

Reply:
I'm still not entirely sure what you mean. Like a try catch?